src/git/unpack.mli

```diff
@@ -720,17 +720,15 @@ module MakeDecoder
 
 module MakeStreamDecoder
     (Hash: S.HASH)
-    (Inflate: S.INFLATE)
-: sig
```

Comment:
you can keep as it was before here, as you already started to cut the line (or put everything on the same line).

Yea I know, someday I should really contribute to `ocamlformat` ...